packages/orbit-components/src/Select/__tests__/Select.test.js

```diff
@@ -1,17 +1,21 @@
 // @flow
 
 import * as React from "react";
-import { shallow, mount } from "enzyme";
+import { render, screen, fireEvent } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import Select from "../index";
 import SPACINGS_AFTER from "../../common/getSpacingToken/consts";
 
-const mockChange = jest.fn();
 const placeholder = "Default placeholder";
 const dataTest = "test";
+const label = "label";
 const tabIndex = "-1";
 const name = "name";
 const id = "test-id";
+const onBlur = jest.fn();
+const onFocus = jest.fn();
+const onChange = jest.fn();
```

Comment:
Move these mocks inside the first `it` block just to be safe, so that it's obvious that they are only being used in the first test. Every test should define their own mocks, and in general be as self-contained as possible.